### ISSUE TITLE
Can't highlight your land and building rights in a “no money” scenario

### ISSUE THREAD

**Issue by Wuzzy2** [OpenRCT2/OpenRCT2#8113]
**OS:** GNU/Linux
**Version:** 0.2.1
**Commit/Build:** 0.2.1

- [X] Reproducible in RCT2 (vanilla)?
- [ ] Specific to multiplayer?

**Steps to reproduce:**
1. Play any “no money” scenario
2. Try to figure out which special building rights you have
3. Fail

## Description

In a “no money” scenario, it's impossible to see your building rights (if you have any) because the land button in the park window has disappeared.
AFAIK the only way to make your special building rights visible (and highlight the land you currently own) is by opening the land buying window from the park window.

## Possible fixes

I can think of two possible ways to fix this:

1. In the view options menu, add a new menu item named “Highlight owned land / building rights”. When activated, all your land and build rights will be highlighted as if the “buy land” window would be open.
2. Always display the “buy land” button, even in “no money” scenarios

Implementing either of these will guarantee that players can always see their owned land and building rights regardless of scenario.

**Comment by Patrik356b** [comment]
i'm in favor for number 1

**Comment by deurklink** [comment]
In my opinion, we could just always display the 'buy land' button. 

If a 'no money' scenario has land for sale, it's just bad scenario design, or someone has activated 'no money' in the cheat menu (possibly with the intention of buying all the land). I did just notice something funny:

1. Go into buy land menu (in forest frontiers, for example)
2. Set scenario to 'no money' using cheat menu
3. Buy a lot of land
4. Game still uses invisible money pool, and you can actually run out of money
![image](https://user-images.githubusercontent.com/6469413/47361911-6f62f580-d6d3-11e8-8999-b0d4b20f9364.png)

**Comment by Wuzzy2** [comment]
> If a 'no money' scenario has land for sale, it's just bad scenario design

I agree, this should not be the case. However, in a *no money* scenario you still might start with building rights, which should be visible.

I also prefer solution number 1.

Hmm, my solution number 2 is not ideal. If you have bought all building rights, but not all land, you can no longer click on the “building rights” button anymore, which means your building rights remain forever invisible. How to reproduce:

- Start scenario with buyable land rights (example: Bumbly Beach from RCT)
- Cheat and give yourself a lot of money
- Buy all land rights
- Close land rights menu
- Open land rights menu again

Now the land rights menu should be open with the “buy land” button being preselected. Since both buttons are grayed-out, you can no longer select the land rights button, thus making the land rights forever invisible to you.

A simple solution would be that while the “buy land” menu is open, highlight BOTH your owned land AND your owned land rights. It's more convenient anyway.

**Comment by Broxzier** [comment]
Duplicate of #2674.

**Comment by Wuzzy2** [comment]
Not exactly a duplicate. This issue is about not being able to *see* your current property, not about “buying” land with infinite money.
This is a subtle difference.

**Comment by lwatson2016** [comment]
Hi, my team would like to work on this issue.
